src/treeview/TreeView.jsx

```diff
@@ -32,6 +32,7 @@ function TreeNodes({ childrenPath, depth, draggable, nodes, parentNode, template
           childrenPath={childrenPath}
           depth={depth}
           draggable={draggable}
+          relativePath={relativePath ? `${relativePath}[${index}]` : `[${index}]`}
```

Comment:
Instead of using a ternary here you could set `""` as the default value for the prop